aspnetcore/azure/devops/cicd.md

```diff
@@ -57,26 +57,26 @@ Remove the local Git deployment with the following steps. VSTS both replaces and
 1. Navigate to the *mywebapp<unique_number>* App Service. As a reminder, the portal's search box can be used to quickly locate the App Service.
 1. Click **Deployment options**. A new panel appears. Click **Disconnect** to remove the local Git source control configuration that was added in the previous chapter. Confirm the removal operation by clicking the **Yes** button.
 
-## Create a VSTS account
+## Create an Azure DevOps
```

Comment:
Should this title end in "account"?

Reply:
should end in "organization -- Create an Azure DevOps organization